src/cmd/build.js

```diff
@@ -45,11 +45,20 @@ export type LocalizedNameParams = {
   manifestData: ExtensionManifest,
 }
 
+// This defines the _locales/messages.json type. See:
+// https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Internationalization#Providing_localized_strings_in__locales
+type LocalizedMessageData = {
+  [messageName: string]: {
+    description: string,
+    message: string,
+  },
+}
+
 export async function getDefaultLocalizedName(
   {messageFile, manifestData}: LocalizedNameParams
 ): Promise<string> {
 
-  let messageData: string | Buffer;
+  let messageData: LocalizedMessageData;
```

Comment:
yeah, I assumed `String.prototype.replace` received a more accurate built-in type with this release but I didn't look at it in depth. Nice detective work! I didn't know about `type-at-pos`.